pkgs/tools/system/logcheck/default.nix

```diff
@@ -1,13 +1,12 @@
 { stdenv, fetchurl, lockfileProgs, perl, mimeConstruct }:
 
 stdenv.mkDerivation rec {
-  _name   = "logcheck";
-  version = "1.3.17";
-  name    = "${_name}-${version}";
+  version = "1.3.18";
+  name    = "logcheck_${version}";
```

Comment:
The hyphen is needed for Nix to know what part of "name" is the package name and what is the version number.